src/main/java/ch/epfl/biop/wrappers/stardist/ij2commands/StarDist2D_SegmentImgPath_Advanced.java

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.biop.wrappers.stardist.ij2commands;
+
+
+import ch.epfl.biop.wrappers.stardist.DefaultStardistTask;
+import ch.epfl.biop.wrappers.stardist.StardistTaskSettings;
+import org.scijava.command.Command;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+import java.io.File;
+
+@Plugin(type = Command.class, menuPath = "Plugins>BIOP>StarDist> StarDist2D... (from file - Advanced)")
+public class StarDist2D_SegmentImgPath_Advanced implements Command {
+
+    @Parameter
+    File image_path;
+
+    @Parameter(style = "directory")
+    File model_path;
+
+    @Parameter(style = "directory")
+    File output_path;
+
+    @Parameter
+    int x_tiles = -1;
+
+    @Parameter
+    int y_tiles = -1;
+
+    @Parameter(style = "format:#.0")
+    float min_norm = (float) 3.0;
+
+    @Parameter(style = "format:#.0")
+    float max_norm = (float) 99.8;
+
+    @Parameter
+    float prob_thresh = -1;
+
+    @Parameter
```

Comment:
Same thing : do you need to display significant digits ? then add a format tag in the style of the parameter